test/control_flow.coffee

```diff
@@ -485,6 +485,293 @@ test "#4267: lots of for-loops in the same scope", ->
   """
   ok CoffeeScript.eval(code)
 
+# Test for issue #2342: Lexer: Inline `else` binds to wrong `if`/`switch`
+test "issue 2343: if / then / if / then / else", ->
```

Comment:
This is very minor, but elsewhere in the tests when we reference an issue we do it as `test "#2343: ...`.